docs/en/setup/Configuration-File.md

```diff
@@ -53,6 +53,8 @@ The `KinD` environment follow these steps:
 1. Wait until all steps are finished and all services are ready with the timeout(second).
 1. Expose all resource ports for host access.
 
+Notice, when you create resources such as webhooks, they need time to set up. If you get some error, you can try `sleep` to wait for the resources established. 
```

Comment:
Is there any command `kubectl` that can wait for webhook ready?

Reply:
I haven't found it yet.